src/lib/components/customer/event/EmptyState.svelte

```
@@ -0,0 +1,55 @@
+<script lang="ts">
+  let { clearSearchUrl } = $props<{ clearSearchUrl: string }>();
+</script>
+
+<div class="empty-state">
+  <div class="empty-icon">🎭</div>
+  <h3 class="empty-title">Không tìm thấy sự kiện nào!</h3>
+  <p class="empty-sub">Hãy thử tìm kiếm với từ khóa khác hoặc khám phá tất cả sự kiện.</p>
+  <a href={clearSearchUrl} class="cta-primary" id="events-clear-search">Xóa tìm kiếm</a>
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)